pallets/gear-program/src/mock.rs

```diff
@@ -40,8 +40,11 @@ type AccountId = u64;
 type BlockNumber = u64;
 type Balance = u128;
 
+#[allow(unused)]
```

Comment:
Annoys me too. Better to `#![allow(unused)]` whole mock mod, because it's only test tooling

Reply:
Done